src/lib/Preview.js

```diff
@@ -462,13 +467,25 @@ class Preview extends EventEmitter {
      */
     download() {
         const { apiHost, queryParams } = this.options;
+        let downloadUrl;
 
         if (checkPermission(this.file, PERMISSION_DOWNLOAD)) {
             // Append optional query params
-            const downloadUrl = appendQueryParams(getDownloadURL(this.file.id, apiHost), queryParams);
+            downloadUrl = appendQueryParams(getDownloadURL(this.file.id, apiHost), queryParams);
             get(downloadUrl, this.getRequestHeaders()).then((data) => {
                 openUrlInsideIframe(data.download_url);
             });
+        } else if (canDownloadWatermarkedRep(this.file)) {
+            // If we can download the watermarked rep but not the original, we assume that the viewer is using the
+            // watermarked rep for preview, so we will download it.
+            // This allows the content URL to be downloaded by the browser
+            const downloadQueryParams = { response_content_disposition_type: 'attachment' };
+            const template = this.viewer.options.representation.content.url_template;
```

Comment:
Will `this.viewer.options.representation` ever be null? One potential example: clicking the download button for a ZIP file that's in a watermarked folder?